stop_compaction_test.py

```diff
@@ -247,3 +247,31 @@ def _grep_log_and_assert(self, node: BaseNode):
 
         self.assertTrue(found_grepped_expression, msg=f'Did not find the expected "{self.GREP_PATTERN}" '
                                                       f'expression in the logs.')
+
+
+class StopCompactionTestICS(ClusterTester):
+
+    def setUp(self):
+        super().setUp()
+        self.node = self.db_cluster.nodes[0]
+        self.storage_service_client = StorageServiceClient(self.node)
+        # insert ~10GB of data
+        populate_data_cmd = "cassandra-stress write cl=QUORUM n=2097152 -schema 'replication(factor=1) " \
+                            "compaction(strategy=IncrementalCompactionStrategy)' -port jmx=6868 -mode cql3 native " \
+                            "-rate threads=80 -pop seq=1..2097152 -col 'n=FIXED(10) size=FIXED(512)' -log interval=5"
+        prepare = self.run_stress_thread(stress_cmd=populate_data_cmd, round_robin=True)
+        self.verify_stress_thread(cs_thread_pool=prepare)
+
+    def test_stop_major_compaction(self):
+        """Verifying data loss on stopping compaction by reading inserted"""
+        self.wait_no_compactions_running()
+        compaction_nemesis = StartStopMajorCompaction(
+            tester_obj=self,
+            termination_event=self.db_cluster.nemesis_termination_event)
+        compaction_nemesis.disrupt()
+        verify_cmd = "cassandra-stress read  cl=QUORUM -schema 'replication(factor=1) " \
```

Comment:
Not clear why using 'quorum' + rf=2. is it a test for only a single node? is there a need for 3 nodes?